src/Components/Todo/Todo.js

```diff
@@ -1,51 +1,91 @@
-import React, { memo, useState, useRef, useEffect } from 'react';
+import React, { memo, useState, useRef, useEffect } from "react";
+import { connect } from 'react-redux';
+import {
+  getTodoEditingId,
+  onEditTodo,
+  markCompletedTodo,
+  removeTodo
+} from '../../store/actions/index';
 
-const Todo = props => {
+const Todo = (props) => {
   const clickInput = useRef(null);
-  const { todo, getTodoEditingId, todoEditingId, onEditedTodo, index, markCompletedTodo, removeTodo } = props;
+  const {
+    todo,
+    getTodoEditingId,
+    todoEditingId,
+    onEditTodo,
+    index,
+    markCompletedTodo,
+    removeTodo,
+  } = props;
   const [text, setText] = useState(todo.text);
   const isEditing = todoEditingId === todo.id;
+  const arr = [1,2];
```

Comment:
1, 2 are magic numbers. You should make it clearer.

- [ ] Add a comment or Assign 1, 2 to constant variables